tests/Middleware/MiddlewareTest.php

```diff
@@ -2,36 +2,43 @@
 
 declare(strict_types = 1);
 
-namespace OpenEuropa\EPoetry\Tests\Middleware;
+namespace OpenEuropa\EPoetry\Tests\Requests;
 
 use GuzzleHttp\Psr7\Response;
-use OpenEuropa\EPoetry\Tests\AbstractTest;
+use OpenEuropa\EPoetry\Serializer\RequestsSerializer;
+use OpenEuropa\EPoetry\Tests\Middleware\MockMiddleware;
 use OpenEuropa\EPoetry\Type\CreateRequests;
-use OpenEuropa\EPoetry\Type\LinguisticRequestIn;
-use OpenEuropa\EPoetry\Type\RequestGeneralInfoIn;
 
 /**
  * @internal
  * @coversNothing
  */
-final class MiddlewareTest extends AbstractTest
+final class MiddlewareTest extends AbstractRequestTest
 {
+    /**
+     * Data provider.
+     *
+     * @return mixed
+     */
+    public function responseParsingCases()
+    {
+        return $this->getFixture('request-test.yml');
```

Comment:
We should rename this fixture to match the test it provides data for, I'd suggest something like: `response-parsing-test.yml`.

Reply:
:+1: